packages/core/lib/package.js

```diff
@@ -1,351 +1,214 @@
 const expect = require("@truffle/expect");
 const TruffleError = require("@truffle/error");
-const Networks = require("./networks");
-const EthPM = require("ethpm");
-const EthPMRegistry = require("ethpm-registry");
-const Web3 = require("web3");
-const { createInterfaceAdapter } = require("@truffle/interface-adapter");
-const async = require("async");
+const { EthPM } = require("ethpm");
```

Comment:
to make this non-breaking, I would suggest creating new packages in the monorepo here, maybe `@truffle/ethpm-v1` and `@truffle/ethpm-v2` and having those packages depend on the respective versions of the ethpm Node package.

Then, you could duplicate and move this `package.js` file to those corresponding new packages